guacamole-ext/src/main/java/org/apache/guacamole/net/auth/Connection.java

```diff
@@ -125,5 +125,17 @@
      *     connection.
      */
     public Set<String> getSharingProfileIdentifiers() throws GuacamoleException;
+    
+    /**
+     * Get the GuacamoleProxyConfiguration for this connection.
```

Comment:
There should be more to the documentation of `X.getFoo()` than "Gets the Foo for this X". There are semantics which apply here that should be noted.

Reply:
Removed.